tools/distrib/python/grpcio_tools/grpcio_tools.bzl

```diff
@@ -34,7 +34,6 @@ def _generate_copied_files_impl(ctx):
                 strip_prefix,
             ))
         destination_path = dest + destination_path[len(strip_prefix):]
-        destination_dir = destination_path.rfind("/")
```

Comment:
@gnossen I removed this as this is not used in the code. And I'd like you to check on this just in case.

Reply:
This should be fine. Thanks for the cleanup.